### ISSUE TITLE
Prompt only for missing args?

### ISSUE THREAD

**Issue by themightychris** [nanovazquez/yargs-interactive#8]
Is there a usage where prompts can be shown only for options not set via command line?

**Comment by nanovazquez** [comment]
Hi @themightychris! There isn't at the moment, but it's not hard to implement. See [here](https://github.com/nanovazquez/yargs-interactive/blob/master/src/yargs-interactive.js#L29), we are filtering options with `prompt` set to `false`. We can add another value to only filter these options if the value is set.

Is that what you are looking for?

**Comment by themightychris** [comment]
@nanovazquez that sounds about right!

**Comment by themightychris** [comment]
If boolean values could be interpreted with something like `typeof prompt == 'boolean' ? (prompt ? 'always' : 'never') : prompt` then it wouldn't need to be a breaking change, right?

**Comment by nanovazquez** [comment]
You are right. But should we? As long as we release a new version I don't see the problem of treating it as a breaking change (unless I'm missing something).

**Comment by nanovazquez** [comment]
We have a candidate in #9. I'll work on the tests and release it as soon as they pass.

**Comment by nanovazquez** [comment]
There you go, @themightychris. The new `prompt` option, with the `if-empty` configuration is now available in **v2.0.0**. You can use it by simply doing something like this:

```js
const options = {
  name: {
    prompt: 'if-empty';
  }
  ...
};
```

And then **yargs-interactive** will prompt the `name` option if a value was not supplied via default value or command parameters.
